package.json

```
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/Brightspace/brightspace-integration",
   "dependencies": {
+    "@brightspace-ui-labs/input-duration": "^1.1.1",
```

We typically don't bother specifying the minor/patch release just to keep things clean. Then the lock file is where the nitty gritty is tracked anyway.

```suggestion
    "@brightspace-ui-labs/input-duration": "^1",
```